src/file-uploader/file-uploader.component.ts

```diff
@@ -42,7 +42,12 @@ const noop = () => {};
 					tabindex="-1"
 					(change)="onFilesAdded()"/>
 				<div class="bx--file-container">
-					<ibm-file *ngFor="let fileItem of files" [fileItem]="fileItem" (remove)="removeFile(fileItem)"></ibm-file>
+					<div *ngFor="let fileItem of files">
+						<ibm-file [fileItem]="fileItem" (remove)="removeFile(fileItem)"></ibm-file>
+						<div *ngIf="fileItem.invalid" class="bx--form-requirement">
+							File size exceeds limit
```

Comment:
This should be provided to the `fileItem`, since size isn't the only reason a file may be invalid